pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1272,4 +1274,28 @@ public void setPreferLaterVersions(boolean preferLaterVersions) {
     public int getWebSocketConnectionsPerBroker() { return webSocketConnectionsPerBroker; }
 
     public void setWebSocketConnectionsPerBroker(int webSocketConnectionsPerBroker) { this.webSocketConnectionsPerBroker = webSocketConnectionsPerBroker; }
+
+    public static ServiceConfiguration convertFrom(PulsarConfiguration conf) throws RuntimeException {
```

Comment:
and should we also add overloaded method `convertFrom(PulsarConfiguration conf, boolean ignoreNonExistMember)` which will not throw `RuntimeException` if the name is different or few of the fields are not present into `ServiceConfiguration` and present into given `PulsarConfiguration`. ??

Reply:
I added the argument `ignoreNonExistMember` and the overloaded method.